examples/event_handling/resample.py

```diff
@@ -1,9 +1,15 @@
 """
-========
-Resample
-========
+===============
+Resampling Data
+===============
+
+
+Downsampling lowers the sample rate or sample size of a signal. In
+this tutorial, the signal is downsampled when the plot is adjusted
+through dragging and zooming.
```

Comment:
... except that the example is completely broken (try it).  It should either be fixed or removed.